release/ray_release/buildkite/filter.py

```diff
@@ -16,14 +18,37 @@ def _unflattened_lookup(lookup: Dict, flat_key: str, delimiter: str = "/") -> An
     return curr
 
 
+def _get_likely_failing_tests() -> List[str]:
+    def _get_last_modified(file):
+        return int(file["LastModified"].strftime("%s"))
+
+    s3 = boto3.client("s3")
+    files = s3.list_objects_v2(
+        Bucket="ray-release-automation-results",
+        Prefix="continuous-release/",
+    )["Contents"]
+    latest_coverage = [file for file in sorted(files, key=_get_last_modified)][-1]
+    data = s3.get_object(
+        Bucket="ray-release-automation-results", Key=latest_coverage["Key"]
+    )
+    return json.loads(data["Body"].read().decode("utf-8"))
```

Comment:
Where do we write this file? Can we maybe add a few comments how this file looks like and how the results look like?

Lastly, can we maybe modularize this a bit. E.g. have a utility in `aws.py` to fetch a file from S3, and then call the utility here? It also seems to me like we should have at least constants defined for the bucket + file location